sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -968,13 +968,13 @@ def expand(self, pcoll):
         return (
             pcoll
             | core.ParDo(self.add_timestamp).with_output_types(
-                Tuple[T, TimestampType])  # type: ignore[misc]
```

Comment:
upgrading to the latest mypy lets us drop these

Reply:
Nice? Does it cause any other issues?

Reply:
No, I was pretty surprised.